pages/index.tsx

```diff
@@ -17,24 +17,26 @@ export default (() => {
     }, []);
 
     return (
-        <div className={`page relative matrix:bg-matrix-bg`}>
-            <NavBar setShowOnboardModal={setShowOnboardModal} />
-            <SwapStore>
-                <Browse />
-            </SwapStore>
-            <UnsupportedNetworkPopup />
-            <OnboardTradeModal
-                onboardStep={onboardStep}
-                setOnboardStep={setOnboardStep}
-                showOnboardModal={showOnboardModal}
-                setShowOnboardModal={() => {
-                    setShowOnboardModal(false);
-                    setTimeout(() => {
-                        setOnboardStep(1);
-                    }, 1000);
-                }}
-            />
+        <>
+            <div className={`page relative matrix:bg-matrix-bg`}>
+                <NavBar setShowOnboardModal={setShowOnboardModal} />
+                <SwapStore>
+                    <Browse />
+                </SwapStore>
+                <UnsupportedNetworkPopup />
+                <OnboardTradeModal
+                    onboardStep={onboardStep}
+                    setOnboardStep={setOnboardStep}
+                    showOnboardModal={showOnboardModal}
+                    setShowOnboardModal={() => {
+                        setShowOnboardModal(false);
+                        setTimeout(() => {
+                            setOnboardStep(1);
+                        }, 1000);
+                    }}
+                />
+            </div>
             <Footer />
-        </div>
+        </>
```

Comment:
This might effect page gradient background added https://www.figma.com/file/gA9jQ5B6ZwWjGKBhGUFbKo/2.0-3.0-Pools?node-id=2411%3A190966